### PR TITLE
Esas/make running state more visible prod 12144

### PR DESCRIPTION
**Pull request by esasova**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

File: src/components/ScenarioParameters/components/ScenarioActions/components/RunningStateSpinner/RunningStateSpinner.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    currentScenarioState === SCENARIO_RUN_STATE.DATA_INGESTION_IN_PROGRESS;
+
+  return isCurrentScenarioRunning ? (
+    <Grid item>
+      <Stack direction="row" spacing={1} sx={{ mr: 2 }}>
+        <CircularProgress color="inherit" size={16} data-cy="running-state-spinner" />
+        <Typography variant="body1" data-cy="running-state-label">
+          {currentScenarioState === SCENARIO_RUN_STATE.RUNNING
+            ? t('commoncomponents.scenariomanager.treelist.node.status.running', 'Running')
+            : t('commoncomponents.scenariomanager.treelist.node.status.dataingestioninprogress')}
```
Comment (csm-thu) [inline src/components/ScenarioParameters/components/ScenarioActions/components/RunningStateSpinner/RunningStateSpinner.js:25]: Can we add a default value for the `dataingestioninprogress` label here?

File: src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
 import React from 'react';
 import { ButtonTesting } from '../../../../../tests/MuiComponentsTesting';
-import { customRender } from '../../../../../tests/utils/renderInThemeAndStoreProviders';
+import { customRender, getByDataCy } from '../../../../../tests/utils';
 import { ScenarioActions } from './';
-import { getByDataCy } from '../../../../../tests/utils';
 import { SCENARIO_RUN_STATE } from '../../../../services/config/ApiConstants';
 import { useFormState } from 'react-hook-form';
 import { useCurrentScenarioState } from '../../../../state/hooks/ScenarioHooks';
+import { TypographyTesting } from '../../../../../tests/MuiComponentsTesting/TypographyTesting';
```
Comment (csm-thu) [inline src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js:12]: There's another import of `ButtonTesting` from the `MuiComponentsTesting` folder, maybe we can group these imports?

File: src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js [hunk 3]
```diff
@@ -141,6 +142,8 @@ describe('Test scenario buttons when scenario is running', () => {
     expect(discardChangesButton.Button).not.toBeInTheDocument();
 
     expect(stopRunButton.Button).toBeVisible();
+    expect(getByDataCy('running-state-spinner')).toBeVisible();
+    expect(runningStateLabel.Typography).toBeVisible();
```
Comment (csm-thu) [inline src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js:146]: We should probably also check that the spinner and 'running' label are not visible (or don't exist) when the scenario is not running

**Review by csm-thu — APPROVED** [review]
Small changes suggested, but LGTM :+1:

File: src/components/ScenarioParameters/components/ScenarioActions/components/RunningStateSpinner/RunningStateSpinner.js (hunk 1 above)
Comment (esasova) [reply src/components/ScenarioParameters/components/ScenarioActions/components/RunningStateSpinner/RunningStateSpinner.js:25]: Nice catch, thanks ! :pray:

File: src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js (hunk 3 above)
Comment (esasova) [reply src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js:146]: Good idea!

File: src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js [hunk 4]
```diff
@@ -92,6 +92,8 @@ describe('Test scenario buttons when scenario is not running', () => {
       expect(saveScenarioButton.Button).not.toBeInTheDocument();
       expect(discardChangesButton.Button).not.toBeInTheDocument();
       expect(stopRunButton.Button).not.toBeInTheDocument();
+      expect(getByDataCy('running-state-spinner')).not.toBeInTheDocument();
+      expect(runningStateLabel.Typography).not.toBeInTheDocument();
```
Comment (nborde-CSM) [inline src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js:96]: Is it really interesting to create a Typography class just for using it as a simple _component.getBydataCy()_ ?
Besides, I do not see any use of _text_

File: tests/MuiComponentsTesting/TypographyTesting.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { getByDataCy } from '../utils';
+
+export class TypographyTesting {
```
Comment (nborde-CSM) [inline tests/MuiComponentsTesting/TypographyTesting.js:6]: See my comment below.

**Review by nborde-CSM — APPROVED** [review]
One comment but LGTM :+1:

File: src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js (hunk 4 above)
Comment (esasova) [reply src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js:96]: I created it for future unit tests that could use a _Typography_ class as it's one our most used components. And _text_ is a remaining of an attempt to test `textContent` but as it's tricky and not so important, I gave up and kept _text_ for future UT :slightly_smiling_face: maybe we need a vote for it, @csm-thu, your opinion ? :smile:

Comment (csm-thu) [reply src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js:96]: No objection to create genreric classes to make UT easier to write, as long as the tested component is used several times. `Typography` seems to be a good candidate for such a generic class :smile:

**Pull request merged**